### DISCUSSION TITLE
What version of python did you use?

### DISCUSSION THREAD

**Discussion by vastakhov** [touhidurrr/iplist-youtube#13] [Q&A] [answered]
What version of python did you use? 
I get the error:
`
  File "/home/user/youtube/iplist-youtube/list_generator.py", line 9
    type IPList = list[IPv4Address | IPv6Address]
         ^^^^^^
SyntaxError: invalid syntax
`

Python 3.10.12

**Comment by touhidurrr** [comment, marked as answer]
**Python 3.12.4**
